include/libtorrent/storage.hpp

```diff
@@ -285,8 +285,7 @@ namespace libtorrent
 		, public storage_piece_set
 		, boost::noncopyable
 	{
-		// hidden
-		storage_interface(): m_settings(0) {}
+		storage_interface() {}
```

Comment:
did you mean to leave this in? If you keep this here, you need to keep the comment too